src/components/TidligstMuligUttaksalder/TidligstMuligUttaksalder.tsx

```diff
@@ -123,6 +125,10 @@ export const TidligstMuligUttaksalder: React.FC<Props> = ({
               }
               values={{
                 ...getFormatMessageValues(intl),
+                nedreAldersgrense: transformAlderToString(
```

Comment:
@Lucas8448 er dette der ikke feil? Skal det vises nedre aldersgrense for denne teksten og ikke 62 år? 
Mente du kanskje å heller ha dynamiske verdier på denne teksten 'omufoeretrygd.readmore.hel.ingress' for å erstatte 67 år med ubetinget-uttaksalder?

Reply:
Jeg ser på PEK-868 at det står kommentar fra Nina om at den skal endres. Da er det nok riktig!